kinto_emailer/tests/test_includeme.py

```diff
@@ -210,6 +210,28 @@ def test_get_messages_can_filter_by_event_class(self):
         assert len(messages) == 1
 
 
+class GroupExpansionTest(unittest.TestCase):
+
+    def test_recipients_are_expanded_from_group_members(self):
+        storage = mock.MagicMock()
+        collection_record = {
+            'kinto-emailer': {
+                'hooks': [{
+                    'template': 'Poll changed.',
+                    'recipients': ['/buckets/b/groups/g'],
```

Comment:
Should we make sure that the collection bucket and the group bucket matches there? To prevent someone from sending mails to another bucket that they don't have permissions about?